openapi.json

```diff
@@ -15198,16 +16049,76 @@
+                    },
+                    {
+                        "label": "TypeScript",
+                        "lang": "JavaScript",
+                        "source": "const environmentOut = await svix.environment.export();"
+                    },
+                    {
+                        "label": "Python",
+                        "lang": "Python",
+                        "source": "environment_out = svix.environment.export()"
+                    },
+                    {
+                        "label": "Python (Async)",
+                        "lang": "Python",
+                        "source": "environment_out = await svix.environment.export()"
+                    },
+                    {
+                        "label": "Go",
+                        "lang": "Go",
+                        "source": "environmentOut, err := svixClient.Environment.Export(ctx)"
+                    },
+                    {
+                        "label": "Kotlin",
+                        "lang": "Kotlin",
+                        "source": "val environmentOut = svix.environment.export()"
+                    },
+                    {
+                        "label": "Java",
+                        "lang": "Java",
+                        "source": "EnvironmentOut environmentOut = svix.getEnvironment().export()"
+                    },
+                    {
+                        "label": "Ruby",
+                        "lang": "Ruby",
+                        "source": "environment_out = svix.environment.export()"
+                    },
+                    {
+                        "label": "Rust",
+                        "lang": "Rust",
+                        "source": "let environment_out = svix.environment().export().await?;"
+                    },
+                    {
+                        "label": "C#",
+                        "lang": "C#",
+                        "source": "var environmentOut = await svix.Environment.ExportAsync()"
+                    },
+                    {
+                        "label": "CLI",
+                        "lang": "Shell",
+                        "source": "svix environment export "
+                    },
+                    {
+                        "label": "cURL",
+                        "lang": "Shell",
+                        "source": "curl -X 'GET' \\\n  'https://api.eu.svix.com/api/v1/environment/export/' \\\n  -H 'Authorization: Bearer AUTH_TOKEN' \\\n  -H 'Accept: application/json' \\\n  -H 'Content-Type: application/json'"
+                    }
                 ]
-            }
-        },
-        "/api/v1/environment/export/": {
+            },
             "post": {
                 "description": "Download a JSON file containing all org-settings and event types",
-                "operationId": "v1.environment.export",
+                "operationId": "v1.environment.export.post",
```

Comment:
The last segment in an operation id is usually something CRUD-ish, right? create/delete/list... that sort of thing. I wonder why `post` is the thing here.

Reply:
My bad. I manually changed this op id here because it was failing to generate the libs due to a repeated op id with the one in line 15964. I changed the other one to be `v1.environment.export.get` so we can leave this one as is; no need to add `.post.` This endpoint is not making a CRUD operation AFAICT, so `create` doesn't look appropriate either.

I'll fix the repeated op id so we don't have to edit this again manually, but I wanted to get this PR unblocked.

cc @svix-aaron1011